### PR TITLE
Retemplate before release

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -199,18 +199,21 @@ $RECYCLE.BIN/
 ### VScode: https://raw.githubusercontent.com/github/gitignore/master/Global/VisualStudioCode.gitignore
 .vscode/*
 
-### Extra Python Items and nccube Specific
+### Extra Python Items and SunPy Specific
```
Comment (DanRyanIrish) [inline .gitignore]: ndcube or SunPy?

File: pyproject.toml [hunk 2]
```diff
@@ -19,17 +19,17 @@ requires = ["setuptools", "wheel"]
   [ tool.sunpy-bot.towncrier_changelog ]
     verify_pr_number = true
     changelog_skip_label = "No Changelog Entry Needed"
-    help_url = "https://github.com/Cadair/sunpy/blob/towncrier/changelog/README.rst"
+    help_url = "https://github.com/sunpy/github/blob/towncrier/changelog/README.rst"
```
Comment (DanRyanIrish) [inline pyproject.toml]: Is this right?

File: pyproject.toml [hunk 3]
```diff
@@ -50,7 +50,7 @@ requires = ["setuptools", "wheel"]
     package = "ndcube"
     filename = "CHANGELOG.rst"
     directory = "changelog/"
-    issue_format = "`#{issue} <https://github.com/sunpy/ndcube/pull/{issue}>`__"
+    issue_format = "`#{issue} <https://github.com/sunpy/github/pull/{issue}>`__"
```
Comment (DanRyanIrish) [inline pyproject.toml]: Is this right?

File: tox.ini [hunk 4]
```diff
@@ -1,41 +1,28 @@
 [tox]
-envlist = py{36,37}-{offline,astropydev,numpydev,sunpydev,sunpyold},build_docs,conda,pycodestyle
+envlist = py{36,37}-{offline,online,astropydev,numpydev,sunpydev},build_docs
```
Comment (DanRyanIrish) [inline tox.ini]: Do we not still want to check codestyle?

File: docs/conf.py [hunk 5]
```diff
@@ -25,26 +25,25 @@
 # Thus, any C-extensions that are needed to build the documentation will *not*
```
Comment (nabobalis) [inline docs/conf.py]: This entire file is screwed. Not sure if its a template issue or what.

**Comment by nabobalis** [comment]
I broke azure.

**Comment by nabobalis** [comment]
I am not sure if more work needs to be done on the template or what really.

**Comment by nabobalis** [comment]
Pydoc seems to keep failing, so the choice is to deal with it somehow.

**Comment by nabobalis** [comment]
It seems the doc tests are not running.

**Comment by nabobalis** [comment]
Ok, now bar the pydoc issue, its all fine now. Doc tests are now run once again.

File: ndcube/mixins/plotting.py [hunk 6]
```diff
@@ -209,7 +205,7 @@ def _plot_2D_cube(self, axes=None, plot_axis_indices=None, axes_coordinates=None
         if axes is None:
             try:
                 axes_coord_check = axes_coordinates == [None, None]
-            except:
+            except BaseException:
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply ndcube/mixins/plotting.py]: autopep8 picked it.

[the remark this answers is not in the document]
Comment (Cadair) [reply ndcube/mixins/plotting.py]: I think `Exception` is probably most appropriate here, autopep8 is probably being very conservative.

Comment (nabobalis) [inline ndcube/mixins/plotting.py]:
```suggestion
            except Exception:
```

Comment (nabobalis) [reply ndcube/mixins/plotting.py]: @Cadair

**Pull request merged**